### ISSUE TITLE
Shutdown option

### ISSUE THREAD

**Issue by A380Coding** [jomjol/AI-on-the-edge-device#1679]
### The Feature

Is a shutdown option required to avoid damage to the sd card while it is being accessed?
Or is it always safe to just cut the power?

**Comment by caco3** [comment]
I agree that it would be wise to close all file handles and sync the filesystem before a restart or removing of the SD-Card.
How ever I currently do not see it as a critical feature. I often remove the card while the system is running or restart it and never had a corrupted filesystem.

**Comment by A380Coding** [comment]
With one of the ESP I had to remformat the sd card. But not sure if this was the issue.
Normally it is running 24/7h
